plugin/user_plugin_server.py

```
@@ -191,164 +401,121 @@ def _register_plugin(plugin: Dict[str, Any]) -> None:
-                logger.warning("plugin.toml %s missing [plugin].id, skipping", toml_path)
+            if not pid: continue
+            
+            entry = pdata.get("entry") # e.g. "plugins.demo:DemoPlugin"
+            if not entry or ":" not in entry: continue
+
+            # 2. 静态导入类 (用于提取元数据，不实例化)
+            module_path, class_name = entry.split(":", 1)
+            try:
+                mod = importlib.import_module(module_path)
+                cls = getattr(mod, class_name)
+            except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)